config/initializers/assets.rb

```diff
@@ -9,3 +9,7 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 Rails.application.config.assets.precompile += %w( search.js )
+
+# Rails.application.config.assets.precompile += %w( '.svg' )  
```

Comment:
I am not entirely sure this is true, but don't want to lose track of it in case this fails in deployment